### ISSUE TITLE
feat(MyAnimeList): Remove ads and unblock 18+ Anime/Manga

### ISSUE THREAD

**Issue by Aiakio** [ReVanced/revanced-patches#363]
### Application

MyAnimeList

### Issue

Contains ads and doesn't show 18+ Anime/Manga.

### Patch

Remove ads and enable 18+ content in the app.

### Motivation

If you are trying to find 18+ Anime/Manga you always need to use the website, because they won't show up in the app.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Over time, a lot of feature requests have been made, which is why I am closing some according to the [contribution guidelines](https://github.com/ReVanced/revanced-patches/blob/main/CONTRIBUTING.md#-submitting-a-feature-request):

> Requests can be accepted or rejected at the discretion of maintainers of ReVanced Patches

This feature request has been closed for one or more of the following reasons:

- Stale for too long
- Low priority
- Maintainers un-interested
- Too complex to realize

Regardless, anyone who comes across this feature request can PR it. Thanks for the feature request!
